src/body/Body.js

```diff
@@ -1,6 +1,7 @@
 import React, {PropTypes, Component} from "react";
 
 import Row from "./Row";
+import {evenRowClassName,oddRowClassName} from "./Row"
```

Comment:
please use one statement to import element and class names from Row:
`import Row, {evenRowClassName, oddRowClassName} from "./Row";`